src/services/ExposureNotificationService/ExposureNotificationService.spec.ts

```diff
@@ -749,6 +749,63 @@ describe('ExposureNotificationService', () => {
     });
   });
 
+  it('calculateNeedsSubmission when monitoring', () => {
+    service.exposureStatus.set({
+      type: ExposureStatusType.Monitoring,
+    });
+    expect(service.calculateNeedsSubmission()).toStrictEqual(false);
+  });
+
+  it('calculateNeedsSubmission when exposed', () => {
+    const today = new OriginalDate();
+    service.exposureStatus.set({
+      type: ExposureStatusType.Exposed,
+      summary: getSummary({
+        today,
+        hasMatchedKey: true,
+        daysSinceLastExposure: 7,
+        attenuationDurations: [20, 0, 0],
+      }),
+    });
+    expect(service.calculateNeedsSubmission()).toStrictEqual(false);
+  });
+
+  it('calculateNeedsSubmission when diagnosed false', () => {
+    service.exposureStatus.set({
+      type: ExposureStatusType.Diagnosed,
+      cycleStartsAt: new OriginalDate('2020-05-18T04:10:00+0000').getDate(),
+      cycleEndsAt: 0,
```

Comment:
`cycleEndsAt` should be 14 days after `cycleStartsAt` to make this more realistic. You could also do it more like the other test you wrote to simulate a cycle that has just finished:
```
      cycleStartsAt: today.getDate() - 15 * ONE_DAY,
      cycleEndsAt: today.getDate() - 1 * ONE_DAY,
```